### ISSUE TITLE
[BUG]: Wave Rally: Murky Water and Corrupt Text

### ISSUE THREAD

**Issue by waspennator** [PCSX2/pcsx2#6554]
### Describe the Bug

On hardware rendering (All backends), the water looks rather murky and dark compared to being bright and clean on software rendering. Also all renderers have some form of corrupt garbage text in some spots where text should be.

### Reproduction Steps

For the text: Just start the game up and watch, garbage text should show up after the first loading bar fills up but before the intro fmvs, after pressing start on the title screen, and when you start a race and text starts popping up on the bottom.

For the Water: Just start a race and look at the water. On software it should look bright and clean and on hardware it will look dark and murky.

### Expected Behavior

The text shouldn't be looking weirdly corrupted, and the water shouldn't be darkly murky on hardware.

### PCSX2 Revision

v1.7.3035

### Operating System

Windows 11

### CPU

Ryzen 5 5600H

### GPU

GTX 1650

### GS Window Screenshots

Text Errors: (Happens across all backends)

![Wave Rally Corrupt Text bug 2](https://user-images.githubusercontent.com/78768499/176817451-1021c270-410f-494d-a052-6a7b269f7c31.png)

![Wave Rally Corrupt text bug 1](https://user-images.githubusercontent.com/78768499/176817465-03582152-9a1d-415a-bae0-2b78759b4fb8.png)

Vulkan Rendering: (Happens on all hardware renderers)

![Wave Rally Murky Water Vulkan](https://user-images.githubusercontent.com/78768499/176817558-63298aa9-c6db-48f4-85c6-42211a4c398d.png)

Software Rendering:

![Wave Rally Murky Water Software](https://user-images.githubusercontent.com/78768499/176817581-784d55d3-e78e-4c5d-b370-4386b02ca03c.png)

### Logs & Dumps

[Wave Rally Corrupt text bug 1.zip](https://github.com/PCSX2/pcsx2/files/9024851/Wave.Rally.Corrupt.text.bug.1.zip)
[Wave Rally Corrupt text bug 2.zip](https://github.com/PCSX2/pcsx2/files/9024852/Wave.Rally.Corrupt.text.bug.2.zip)
[Wave Rally Murky Water Software Bug.zip](https://github.com/PCSX2/pcsx2/files/9024853/Wave.Rally.Murky.Water.Software.Bug.zip)
[Wave Rally Murky Water Vulkan bug.zip](https://github.com/PCSX2/pcsx2/files/9024856/Wave.Rally.Murky.Water.Vulkan.bug.zip)

**Comment by lightningterror** [comment]
Can you test if Preload Frame Data hw hack helps?

**Comment by waspennator** [comment]
Tried that, it doesn't do anything with the text, but it changes the water a bit. Still isn't correct though.

![Wave Rally Water Preload Frame Data](https://user-images.githubusercontent.com/78768499/176975357-6f09a2b6-c07b-4319-85ef-15bec92a1f68.png)
[Wave Rally Vulkan water bug with Preload Frame Data.zip](https://github.com/PCSX2/pcsx2/files/9031925/Wave.Rally.Vulkan.water.bug.with.Preload.Frame.Data.zip)

**Comment by coornio** [comment]
The first pictured text bug happens only in software when tested from the dump.
The second pictured text bug appears unfixable, regardless of setting or backend.

I also tried the (original water bug) dump with PFD on and it appears to work there. 
![gs_20220702173511_Wave Rally_SLUS-20313](https://user-images.githubusercontent.com/11655762/177005136-00a0ce56-c03e-4604-88bb-9a29f4307b9f.png)

When I tried the new dump you took with PFD (and enabled PFD myself for it though), the result matched your own screenshot.

I wonder if this might be a texture cache issue. If so, dumps will be useless to troubleshoot it..
